src/Adapter/Product/Stock/CommandHandler/UpdateProductStockInformationHandler.php

```diff
@@ -49,17 +50,23 @@ final class UpdateProductStockInformationHandler implements UpdateProductStockIn
      * @var MovementReasonRepository
      */
     private $movementReasonRepository;
+    /**
```

Comment:
```suggestion
    /**
```
Not to block a PR, but in case you have any other feedback, could fix it together.